05-dom-events/exercise/08-sudomu/solution.js

```diff
@@ -25,22 +25,21 @@ function solve() {
     }
     let isValid = false;
 
-    for (let x = 0; x < columnsWithRows.length; x++) {
-      if (
-        columnsWithRows[x][0] != columnsWithRows[x][1] &&
-        columnsWithRows[x][0] != columnsWithRows[x][2] &&
-        columnsWithRows[x][1] != columnsWithRows[x][2] &&
-        columnsWithRows[0][x] != columnsWithRows[1][x] &&
-        columnsWithRows[0][x] != columnsWithRows[2][x] &&
-        columnsWithRows[1][x] != columnsWithRows[2][x]
-      ) {
-        isValid = true;
-      } else {
+ let sum = columnsWithRows[0].reduce((a, b) => a + b);
```

Comment:
Why `let`? We are not modifying this variable anywhere, so I suggest using const here